sklearn/linear_model/tests/test_logistic.py

```diff
@@ -442,16 +442,24 @@ def test_ovr_multinomial_iris():
     train, target = iris.data, iris.target
     n_samples, n_features = train.shape
 
-    # Use pre-defined fold as folds generated for different y
-    cv = StratifiedKFold(target, 3)
-    clf = LogisticRegressionCV(cv=cv)
+    # The cv splits from stratified kfold (where stratification is done based
+    # on the fine-grained iris classes, i.e, before the classes 0 and 1 are
+    # conflated) is used for both clf and clf1
```

Comment:
I like how this test was commented overall. I'd rephrase `cv splits` to `cv indices` because splits looks like a verb here.